spark/src/main/java/org/infinispan/tutorial/simple/spark/SimpleSparkJob.java

```diff
@@ -0,0 +1,89 @@
+package org.infinispan.tutorial.simple.spark;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaDoubleRDD;
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.Configuration;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.spark.rdd.InfinispanJavaRDD;
+
+import java.io.Serializable;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Properties;
+
+public class SimpleSparkJob {
+
+   public static void main(String[] args) throws UnknownHostException {
+      // Obtain the local address
+      String localAddress = InetAddress.getLocalHost().getHostAddress();
+
+      // Adjust log levels
+      Logger.getLogger("org").setLevel(Level.WARN);
+
+      // Create the remote cache manager
+      Configuration build = new ConfigurationBuilder().addServer().host(localAddress).build();
```

Comment:
I like this local address thing, maybe I should change my remote / scripting examples too